stream.go

```diff
@@ -315,12 +315,7 @@ func (s *Stream) transcode(startId int) {
 		scale = fmt.Sprintf("scale_vaapi=w=%d:h=%d:force_original_aspect_ratio=decrease", s.width, s.height)
 	} else if CV == "h264_nvenc" {
                 // NVENC
-		format = "format=nv12|cuda,hwupload"
-                if s.width >= s.height {
-                        scale = fmt.Sprintf("scale_cuda=-2:%d", s.height)
-                } else {
-                        scale = fmt.Sprintf("scale_cuda=%d:-2", s.width)
-                }
+		scale = fmt.Sprintf("scale_cuda=w=%d:h=%d:force_original_aspect_ratio=decrease:passthrough=0", s.width, s.height)
```

Comment:
Are you sure you don't need `format`? GPUs might not support decoding everything, e.g. AV1

Reply:
Good point! I'll add that back in...

Reply:
I'd gotten it into my head that it's there due to QSV working with system RAM and whatnot.

While I'm at it I'll still have to test AV1 encoded videos so that I can verify whether CPU-decoding + GPU-encoding is working.